docs/micro-rdk/movement-sensor/gyro-mpu6050.md

```diff
@@ -0,0 +1,121 @@
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand
+---
+
+The `gyro-mpu6050` movement sensor model supports a combination [gyroscope and accelerometer manufactured by TDK InvenSense](https://invensense.tdk.com/products/motion-tracking/6-axis/mpu-6050/).
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
+
+Navigate to the **Config** tab of your robot's page in [the Viam app](https://app.viam.com).
+Click on the **Components** subtab and click **Create component**.
+Select the `movement-sensor` type, then select the `gyro-mpu6050` model.
+Enter a name for your movement sensor and click **Create**.
+
+{{< imgproc src="/components/movement-sensor/mpu6050-builder.png" alt="Creation of an `gyro-mpu6050` movement sensor in the Viam app config builder." resize="600x" >}}
+
+Copy and paste the following attribute template into your movement sensor's **Attributes** box.
+Then remove and fill in the attributes as applicable to your movement sensor, according to the table below.
+
+{{< tabs >}}
+{{% tab name="Attributes template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "board": "<your-board-name>",
+  "i2c_bus": "<your-i2c-bus-name-on-board>",
+  "use_alt_i2c_address": <boolean>
+}
+```
+
+{{% /tab %}}
+{{% tab name="Attributes example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "board": "local",
+  "i2c_bus": "default_i2c_bus"
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /tab %}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "<your-sensor-name>",
+      "model": "gyro-mpu6050",
+      "type": "movement_sensor",
+      "namespace": "rdk",
+      "attributes": {
+        "board": "<your-board-name>",
+        "i2c_bus": "<your-i2c-bus-name-on-board>",
+        "use_alt_i2c_address": <boolean>
+      },
+      "depends_on": []
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "name": "local",
+      "model": "esp32",
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {
+        "pins": [15, 21, 22],
+        "i2cs": [
+          {
+            "name": "default_i2c_bus",
+            "bus": "i2c0",
+            "data_pin": 21,
+            "clock_pin": 22
+          }
+        ]
+      }
+    },
+    {
+      "name": "my_accelgyro",
+      "model": "gyro-mpu6050",
+      "type": "movement_sensor",
+      "namespace": "rdk",
+      "attributes": {
+        "board": "local",
+        "i2c_bus": "default_i2c_bus"
+      }
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## Attributes
+
+<!-- prettier-ignore -->
+| Name                  | Type    | Inclusion    | Description |
+| --------------------- | ------- | ------------ | ----------- |
+| `board`               | string  | **Required** | The `name` of the [board](/components/board/) to which the device is wired. |
```

Comment:
As above, should all of these be more specific, for example the following?
```suggestion
| `board`               | string  | **Required** | The `name` of the [ESP32 board](/micro-rdk/board/esp32) to which the device is wired. |
```

Reply:
I actually considered this but decided not to as I figure that when the micro-RDK is expanded to include more types of boards we would have to change it back anyways. That ok?

Reply:
Hmm that makes sense but then how about this, so that users don't end up in main RDK board land and get confused?
```suggestion
| `board`               | string  | **Required** | The `name` of the [board](/micro-rdk/board/) to which the device is wired. |
```